README.md

```diff
@@ -43,6 +43,10 @@ The filename-modifier applied to each buffer name. Default is `':.'`.
 
 The indicator to use for a modified buffer. Default is `'+'`.
 
+##### `g:lightline#bufferline#more_buffers`
+
+The indicator to use when there are buffers that are not shown on the bufferline because they didn't fit the available space. Default is `…`.
```

Comment:
I would prefer to use `...` as default since it is available in all fonts and in the ASCII range. A user can set it to `…` in their vim config if their font supports it.

Reply:
Sure - done :+1: